src/main/scala-2.11/markets/clearing/engines/CDAMatchingEngine.scala

```diff
@@ -84,28 +83,28 @@ class CDAMatchingEngine(askOrdering: PriceOrdering[AskOrder],
         val price = formPrice(incoming, askOrder)
 
         if (residualQuantity < 0) {  // incoming order is smaller than existing order
-          val (filledOrder, residualOrder) = askOrder.split(-residualQuantity)
-          val totalMatch = TotalMatch(filledOrder, incoming, price)
-          _askOrderBook.add(residualOrder)  // SIDE EFFECT!
-          matches.enqueue(totalMatch)
+          val (_, residualAskOrder) = askOrder.split(-residualQuantity)
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Field name does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1545342681/issues/source?bid=2921361&fileBranchId=3015559#l86)